### DISCUSSION TITLE
External territory payment notifications

### DISCUSSION THREAD

**Discussion by ekzyis** [stackernews/stacker.news#3027] [Ideas]
**Is your feature request related to a problem? Please describe.**

Car mentioned a while ago that he would like to receive an email when his payment is due since he tends to forget to pay the rent. I second this. I also tend to forget to make my monthly payments, especially because ~oracle is not maintained by my main account (keeping track of the payment notification is now easier thanks to #644 though).

We currently only show a red notification every day the payment is due but since it's not sticky, it can easily get buried under other notifications, especially if they happen after the daily system notifications (rewards, revenue etc.)

**Describe the solution you'd like**

Have `notify me when ...` settings for due territory payments that you can configure per method:

mobile:

![localhost_3000_settings(iPhone SE)](https://github.com/user-attachments/assets/9f918b26-e9ba-4daf-8ad2-c053dd6c4d18)

desktop:

![2024-09-19-194320_448x384_scrot](https://github.com/user-attachments/assets/3b77f581-2369-4af3-9e77-121049f1afcc)

**Describe alternatives you've considered**

make system notification at least stick to the top of notifications.

**Additional context**

n/a

**Comment by huumn** [comment]
I don't think the UI is right for this (although I respect the laziness). Otherwise I think this makes sense.
